main.tf

```diff
@@ -22,4 +22,14 @@ data "aws_eip" "nat_ips" {
 
 locals {
   use_existing_eips = length(var.existing_nat_ips) > 0
+  map_map = {
```

Comment:
looks like this `map_map` will be included in all modules.
Maybe we should move it to `terraform-aws-utils`?

Reply:
`map_map` is not my first choice, but whatevs. ;)